src/server/services/procedures/roboscape/ciphers.js

```diff
@@ -1,4 +1,4 @@
-const Speck32 = require('./speck.js');
+const Speck32 = require('./speckjs/src/ciphers/speckNative.js');
```

Comment:
Why has this been changed? I suspect this was an accident since it doesn't seem related to any of the other changes.